src/web/lib/components/ThemeColorsEditor/index.scss

```diff
@@ -20,7 +20,7 @@
   padding: $grid * 4;
   position: relative;
   flex: 1 1 640px;
-  height: 320px;
+  height: 400px;
```

Comment:
Make it 350px. Then there is enough space around the storage-space-information element to look appealing in both the Custom and Advanced Colors tab.